GNUmakefile

```diff
@@ -3,6 +3,9 @@ GRAFANA_VERSION ?= latest
 testacc:
 	TF_ACC=1 go test ./... -v $(TESTARGS) -timeout 120m
 
+testacc-enterprise: testacc
+	TF_ACC=1 go test ./... -tags="enterprise" -v $(TESTARGS) -timeout 120m
```

Comment:
Great idea using a build tag. It'd be nice to add this to CI eventually though too. Right now we run the acceptance tests without enterprise enabled. Not sure how hard it would be to get an enterprise enabled container in our Drone build.